### PR TITLE
[Fix] accessory button and help text color is not clearly visible

### PR DESCRIPTION
**Pull request by des12437**
## Description

Fixes a bug in iOS 16+ where the help text and accessory button are not clearly visible when the list item selection color is the accent color (purple/pink) instead of grey. This PR adds opacity to the help text so that the system sets the text color. The accessory button is now a label with an icon and tap gesture rather than a Button since the system will set the color automatically.

## Linked Issue(s)

- `swift/issues/3878`

## How To Test
- Test on iPhone and iPad on iOS 16 and iOS 17

## Screenshots
Before:

After:

### CODE REVIEW

**Review by saratchandrakarumuri — DISMISSED** [review]
Tested on both iOS 15 and 16 platforms and it is now rendering as expected on both the platforms

File: Shared/Supporting Files/Views/SampleRow.swift [hunk 1]
```diff
@@ -32,18 +32,18 @@ struct SampleRow: View {
...
                         .transition(.move(edge: .top).combined(with: .opacity))
                 }
             }
             Spacer()
-            Button {
-                isShowingDescription.toggle()
-            } label: {
+            Label {} icon: {
                 Image(systemName: "info.circle")
                     .symbolVariant(isShowingDescription ? .fill : .none)
```
Comment (yo1995) [inline Shared/Supporting Files/Views/SampleRow.swift:42]:
```suggestion
                    .symbolVariant(isShowingDescription ? .fill : .none)
                    .imageScale(.medium)
```

**Review by yo1995 — COMMENTED** [review]
Just one suggestion, otherwise looks good.

**Review by yo1995 — APPROVED** [review]

**Comment by yo1995** [comment]
@saratchandrakarumuri in case you missed the email notification 😉

**Review by saratchandrakarumuri — APPROVED** [review]
Tested again with iOS 16 and iOS 17. This is now rendering as expected

**Pull request merged**